test/lib/cql_test_env.cc

```diff
@@ -11,6 +11,7 @@
 #include <random>
 #include <seastar/core/thread.hh>
 #include <seastar/util/defer.hh>
+#include "auth/authenticator.hh"
```

Comment:
Is this necessary?

Reply:
Remnant of the previous version. I'll remove it